src/coreclr/jit/gentree.h

```diff
@@ -4022,12 +4022,8 @@ const char* getWellKnownArgName(WellKnownArg arg);
 struct CallArgABIInformation
 {
     CallArgABIInformation()
-<<<<<<< HEAD
         : ArgNum((unsigned)-1)
         , NumRegs(0)
```

Comment:
This does not look correct. Upstream deleted `ArgNum` and so should we. If there are dependencies on it on `TARGET_WASM` code they should be deleted too.

Reply:
Thanks, removed `ArgNum` now.